tests.html

```diff
@@ -104,6 +104,30 @@ <h6 id="6">6</h6>
 	</table>
 </section>
 
+<section>
+	<h1>blockquote</h1>
+
+	<table class="reftest" data-test="innerHTML">
+		<tr title="md-block">
+			<td>
+				<md-block>
+					> Line one
+					> Line two
+					> Line three
+
```

Comment:
Great that you added tests! In addition to having tests that verify quotes work, we should also have tests that verify `>` *outside* of quotes also works.

Reply:
Added some non-happy path tests

Reply:
I would split some of these into separate tests. Unless you're doing integration testing, you want to isolate effects  as much as possible. E.g. imagine if `>` used literally only worked properly iff there was a blockquote earlier in the text. This test wouldn't catch that!

Reply:
Nice, makes sense, split and added some test cases